apps/web/screens/Article/Article.tsx

```diff
@@ -492,7 +492,7 @@ const ArticleScreen: Screen<ArticleProps> = ({
         printHidden
       >
         {/**
```

Comment:
nitpick: maybe remove the {/** and */} 
since lines below are using //
;D

Reply:
This was there previously. I haven't altered this (my ide removed extra spaces around it). But I can remove it